pre_commit/util.py

```diff
@@ -36,10 +36,11 @@
 Dumper = getattr(yaml, 'CSafeDumper', yaml.SafeDumper)
 
 
-def yaml_dump(o: Any) -> str:
+def yaml_dump(o: Any, style: Optional[str] = None) -> str:
```

Comment:
this should be named-only if anything and match the name of the argument from yaml